packages/core/core/src/BundleGraph.js

```diff
@@ -860,7 +896,8 @@ export default class BundleGraph {
     }
 
     return this._graph
-      .findAncestors(node, node => node.type === 'dependency')
+      .getNodesConnectedTo(node)
```

Comment:
Doesn't this cause e.g. child bundle entries to have no incoming dependencies because the dependency itself points to the bundle(group)?

Reply:
Nothing seems to be depending on that right now. Would you say that's expected behavior?

Reply:
getIncomingDependencies is used in very few places at the moment, but I would say it's expected behavior. Otherwise, there is no way to get the incoming deps for a child bundle entry via the public `BundleGraph`.